mne/preprocessing/artifact_detection.py

```diff
@@ -162,11 +164,13 @@ def annotate_movement(raw, pos, rotation_velocity_limit=None,
     """
     sfreq = raw.info['sfreq']
     hp_ts = pos[:, 0].copy()
-    hp_ts -= raw.first_samp / sfreq
+    orig_time = raw.info['meas_date']
+    if orig_time is None:
+        hp_ts -= raw.first_samp / sfreq
```

Comment:
@sappelhoff this was the key change. We only need to shift the `head_pos` time reference by `first_samp` if times are absolute / `orig_time is not None`

Reply:
ahhhh yes, makes sense! Thanks for finding that!

I just think you are missing a "not" in your sentence? I.e., we only need to shift if times are NOT absolute / `orig_time is None`?